cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-cli-guide.adoc

```diff
@@ -0,0 +1,426 @@
+----
+
+The cluster state will change from “waiting” to “installing” to "ready". This will take about 40 minutes.
+
+. Once the state changes to “ready” your cluster is installed.
+
+== Manual Mode
+If you want to review the roles and policies before applying them to a cluster, use the manual method. This method requires running a few extra commands to create the roles and policies.
+
+In this section we will make use of the `--interactive` mode.  See the documentation on xref:../../../rosa_install_access_delete_clusters/rosa-sts-interactive-mode-reference.adoc#rosa-sts-interactive-mode-reference[interactive mode] for a description of the fields in this section.
+
+=== Creating account roles
+. If this is the _first_ time you are deploying ROSA in this account and you have _not_ yet created the account roles, create the account-wide roles and policies, including the Operator policies. The command creates the needed JSON files for the required roles and policies for your account in the current directory. It also outputs the `aws` CLI commands that you need to run in order to create these objects.
++
+Run the following command to create the needed files and output the additional commands:
++
+[source,terminal]
+----
+rosa create account-roles --mode manual
+----
++
+.Example output
+[source,terminal]
+----
+I: All policy files saved to the current directory
+I: Run the following commands to create the account roles and policies:
+aws iam create-role \
+--role-name ManagedOpenShift-Worker-Role \
+--assume-role-policy-document file://sts_instance_worker_trust_policy.json \
+--tags Key=rosa_openshift_version,Value=4.8 Key=rosa_role_prefix,Value=ManagedOpenShift Key=rosa_role_type,Value=instance_worker
+aws iam put-role-policy \
+--role-name ManagedOpenShift-Worker-Role \
+--policy-name ManagedOpenShift-Worker-Role-Policy \
+--policy-document file://sts_instance_worker_permission_policy.json
+----
+
+. Check the contents of your current directory to see the new files. We will use the `aws` CLI to create each of these objects.
++
+.Example output
+[source,terminal]
+----
+$ ls
+openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy.json
+sts_instance_controlplane_permission_policy.json
+openshift_cluster_csi_drivers_ebs_cloud_credentials_policy.json        sts_instance_controlplane_trust_policy.json
+openshift_image_registry_installer_cloud_credentials_policy.json          sts_instance_worker_permission_policy.json
+openshift_ingress_operator_cloud_credentials_policy.json                 sts_instance_worker_trust_policy.json
+openshift_machine_api_aws_cloud_credentials_policy.json                   sts_support_permission_policy.json
+sts_installer_permission_policy.json                                      sts_support_trust_policy.json
+sts_installer_trust_policy.json
+----
+
+. *Optional:* Open the files to review what you will create. For example if we open the `sts_installer_permission_policy.json` we can see:
```

Comment:
```suggestion
. *Optional:* Open the files to review what you will create. For example, if we open the `sts_installer_permission_policy.json` we can see:
```

Reply:
Also not a big fan of using "we" in this steps, but not sure if needs to be corrected in these tutorials?